### PR TITLE
Change the onelive directory structure

### LINKED ISSUE #621: Change the onelive directory structure
**Is your feature request related to a problem? Please describe.**
- Need a new file structure for onelive

**Describe the solution you'd like**
- Move the images from the images folder.
- Create `archive` directory as shown.

**Describe alternatives you've considered**
Screenshot from 2020-03-24 10-20-42

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #621 

## Goals

- Change the onelive directory structure

## Approach

- Move the images from the images folder.
- Create archive directory as shown.
Screenshot from 2020-03-24 10-20-42

  
### Preview Link

https://pr-622-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/onelive/index.html [hunk 1]
```diff
@@ -275,50 +277,4 @@ <h5 class="title">
     </div>
 </script>
 
-<script>
-    $(function () {
-        loadNavAndFooter('assets/content/static');  //relative path to content directory
-        loadProfiles(); // call function to load profiles
-        loadFaqComponent();
-    });
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/index.html]: Keep these line here

File: src/main/webapp/onelive/archive/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/index.html]: EOF

File: src/main/webapp/onelive/functions.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                $(this).children("div").children("i").addClass("fa-minus");
+                $('.card-toggle').removeClass("card shadow");
+                $(this).parent("div").parent("div").addClass("card shadow");
+            }else{
+                $(this).parent("div").parent("div").removeClass("card shadow");
+                $(this).children("div").children("i").removeClass("fa-minus");
+                $(this).children("div").children("i").addClass("fa-plus");
+            }
+        }
+    );
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/functions.js]: This logic is available in the main functions.js component. isn't it?

File: src/main/webapp/onelive/index.html (hunk 1 above)
Comment (jayasanka-sack) [reply src/main/webapp/onelive/index.html]: Keep Ready function on the page itself.

File: src/main/webapp/onelive/functions.js (hunk 3 above)
Comment (jayasanka-sack) [reply src/main/webapp/onelive/functions.js]: `loadFaqComponent();` as I can remember.

File: src/main/webapp/onelive/styles.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.faq .accordion .qa-card {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+.faq .accordion .question .question-heading {
+    padding-right: 30px;
+}
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/styles.css]: Do not duplicate css styles. or js function. 

Reuse as much as possible.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — APPROVED** [review]
Great!

**Review by Gravewalker666 — APPROVED** [review]

**Pull request merged**